### PR TITLE
Adding prediction using Decision Tree Regression

### PR DESCRIPTION
**Pull request by harshita-2402**
Fixes issue #27 

Done prediction of daily Covid cases using Decision Tree Regression.
Used the DecisionTreeRegressor from the sklearn.tree library for the prediction along with Pandas, NumPy and Matplotlib libraries for data preprocessing and visualization.

The net accuracy on the test set : 98.361%

### CODE REVIEW

**Comment by SagarDhandare** [comment]
@harshita-2402 Everything will be fine but at last, when you are using Decision Tree Regressor algorithm, you haven't imported the DecisionTreeRegressor model from sci-kit learn. Just do one thing make a change and upload it again. 🙂

**Comment by harshita-2402** [comment]
@SagarDhandare Oh, okay. Thanks for pointing it out.

**Comment by SagarDhandare** [comment]
@harshita-2402 Looking Good To Me 😀
@ayeshafalak Can you please review it. 🙂
